checker/subtyping.ml

```diff
@@ -222,9 +214,9 @@ let check_inductive  env mp1 l info1 mib2 spec2 subst1 subst2=
     | FakeRecord, FakeRecord -> true
     | PrimRecord info1, PrimRecord info2 ->
       let check (id1, p1, pb1) (id2, p2, pb2) =
-        Id.equal id1 id2 &&
-        Array.for_all2 Constant.UserOrd.equal p1 p2 &&
-        Array.for_all2 eq_projection_body pb1 pb2
+        (* we don't care about the id, the types are inferred from the inductive
+           (ie checked before now) *)
```

Comment:
This check is only in theory for now, see #7909.

Reply:
Ok.